frontend/src/app/my-orders/my-orders.component.ts

```diff
@@ -14,8 +14,10 @@ export class MyOrdersComponent implements OnInit {
 
   ngOnInit(): void {
     this.orderControllerService.getMyOrders().subscribe(myOrders => {
-      this.orders = myOrders;
-    });
-  }
+      this.orders = myOrders.filter(s => {
+        return new Date(s.lastUpdate!.toString()).getTime() + 86400000 > new Date().getTime() && s.status == 'SHIPPED' || s.status!='SHIPPED'
```

Comment:
I would suggest a cleanup here: extracting the two dates into variables would make the code cleaner:

```
let now = new Date().getTime();
let oneDayFromLastUpdate = new Date(s.lastUpdate!.toString()).getTime() + 86400000;
```

One letter variable names are usually discouraged. Could you please replace the `s` on line 17 with something like `myOrder` ?

I find the last two predicates interesting: 

```
s.status == 'SHIPPED' || s.status!='SHIPPED'
```

As far as I can see, this is always true. This can also be a javascript-related trick, some play with `==` and `===`, I am not sure. If it is a bug, could you please fix it? If it is a js-trick, would you please let me know?

Reply:
Super, thank you!